tests/Doctrine/Tests/Common/Cache/MongoDBCacheTest.php

```diff
@@ -6,20 +6,34 @@
 use Doctrine\Common\Cache\MongoDBCache;
 use MongoClient;
 use MongoCollection;
+use MongoConnectionException;
 
 /**
  * @requires extension mongodb
  */
 class MongoDBCacheTest extends CacheTest
 {
+    private static $skip;
+
     /**
      * @var MongoCollection
      */
     private $collection;
 
     protected function setUp()
     {
-        $mongo = new MongoClient();
+        if (static::$skip === true) {
```

Comment:
This is nasty, I known... the idea is to speed up the failure.

@Ocramius @alcaeus I'd love if you could provide a better approach.